src/gsi/stpt.f90

```diff
@@ -233,9 +233,9 @@ subroutine stpt(thead,dval,xval,out,sges,nstep,rpred,spred)
                  ts_prime=tt(kk)
                  tg_prime=valsst2+sges(kk)*valsst
                  qs_prime=valq2+sges(kk)*valq
-                 us_prime=valu2+sges(kk)*val
-                 vs_prime=valv2+sges(kk)*val
-                 psfc_prime=val2+sges(1)*val
+                 us_prime=valu2+sges(kk)*valu
```

Comment:
Were these three lines errors in the original code? valv and valu are not used in the original code.

Reply:
Yes.  This was an error in the original code that I fixed.